Pod/VOKNavigationHelper/VOKNavigationHelper.h

```diff
@@ -0,0 +1,45 @@
+//
+//  VOKNavigationHelper.h
+//  VOKUtilities
+//
+//  Created by brock Boland on 1/24/17.
+//  Copyright (c) 2017 Vokal. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface VOKNavigationHelper : NSObject
+
+/**
+ * Replace the root view controller of the app delegate's window with the provided view controller.
+ * This method will make all subviews on the window remove themselves from their superview, and
+ * dismiss any presented view controller, to ensure that the view stack is completely clear before
+ * setting the rootViewController on the window.
+ *
+ * @param viewController  View controller to display
+ */
++ (void)clearExistingViewsAndSwitchToViewController:(UIViewController *)viewController;
```

Comment:
Do you want to provide an `animated:(BOOL)animated` parameter, or a version that takes such a parameter?